examples/src/keyrings/set_encryption_algorithm_example.py

```diff
@@ -0,0 +1,151 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example demonstrates how to set an encryption algorithm while using the Raw AES Keyring
```

Comment:
Can we call these "algorithm suites" instead of "algorithm" or "encryption algorithm"?
It's more accurate, and is consistent with docs.

Reply:
Yes